app/models/user.rb

```diff
@@ -1,3 +1,17 @@
 class User < ApplicationRecord
-    has_secure_password
+  has_secure_password
+
+  # Will return an array of follows for the given user instance
+  has_many :received_follows, foreign_key: :followed_user_id, class_name: "Follow"
```

Comment:
@idprietop what is the difference between **received_follows** and **followers**

Reply:
@idprietop you meant the output is different te first one returns multiple instances of the Follower model and the other one returns a list of Users, am I right?

Reply:
Yes, exactly that.